specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json

```diff
@@ -1605,6 +1605,7 @@
       "in": "query",
       "required": false,
       "description": "The number of result items to skip.",
+      "default": 0,
```

Comment:
Why does `SkipQueryParameter` have `default: 0`, but `TopQueryParameter` does not?  And what caused this to change in this version upgrade?

Reply:
Well, because the default is logically to skip no records unless a skip value is provided, so this change is essentially documentation that should have no impact.  Setting a default for 'top'  would mean limiting the number of records by default.  Generally the default for top is taken as 'all records', which doesn't have a specific numeric value.